src/tools/haxelib/Data.hx

```diff
@@ -38,6 +38,7 @@ typedef UserInfos = {
 typedef VersionInfos = {
 	var date : String;
 	var name : String;
+	var downloads : Int;
```

Comment:
Right, because they're anonymous objects serialization can call `setField` even if the field isn't known to the client.  Makes sense, thanks.